rules.ini

```diff
@@ -43,6 +43,7 @@ ChromeEngine[] = (?:^|/)ChromeEngine\d\.dll$
 ChromeEngine[] = \.rpack$
 Clausewitz[] = (?:^|/)common/defines\.(?:lua|txt)$
 Clausewitz[] = ^(?:game/)?common/defines/00_defines\.txt$
+ClickTeamFusion = /mmfs2\.dll
```

Comment:
Use (?:^|/)

Reply:
The DLL is *always* in a subfolder. Should I make the change regardless?

Reply:
Yes because there's a little optimization we do for this.

Reply:
Done.